views/editQuiz.handlebars

```diff
@@ -0,0 +1,36 @@
+<body>
+    {{> header}}
+
+    <div class="edit_quiz">
+        <div class="edit_intro">
+            <input placeholder="Title..."></input>
+            <textarea placeholder="Description..."></textarea>
+            <div class="edit_quiz_tags">
+                <p>Tags (separated by space)</p>
+                <input placeholder="Tags..."></input>
+            </div>
+        </div>
+        <div class="edit_question">
+            <p>Edit Questions</p>
+        </div>
+        <div class="edit_questions_container">
+            <div class="edit_flash_card">
+                <button type="button" class="remove_card_button" class="clickable">x</button>
+                <textarea class="edit_quiz_question_input" placeholder="Question..."></textarea>
+                <textarea class="edit_quiz_answer_input" placeholder="Answer..."></textarea>
+            </div>
+            <div class="edit_flash_card">
```

Comment:
Please make these partials and set the code up to compile them. We will need to bring in these edit cards every time the users clicks add in the client-side js. It is ok to leave these here for examples for now.